web_dashboard.py

```
@@ -0,0 +1,710 @@
+        config = try_load_yaml(MAIN_HERMES_CONFIG)
+        gbrain = config.get("mcp_servers", {}).get("gbrain", {})
+        url = gbrain.get("url")
+        headers = gbrain.get("headers", {}) if isinstance(gbrain.get("headers"), dict) else {}
+        sanitized_url = sanitize_url(url)
+        repo_exists = BRAIN_ROOT.exists()
+        endpoint_reachable = False
+        http_status: int | None = None
+        last_error: str | None = None
+
+        if url:
+            request = Request(url, headers=headers or {}, method="GET")
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)